tfhe/src/high_level_api/integers/keys.rs

```diff
@@ -25,38 +25,22 @@ impl IntegerConfig {
         }
     }
 
-    pub(in crate::high_level_api) fn all_default() -> Self {
-        Self::default_big()
-    }
-
-    pub(in crate::high_level_api) fn all_none() -> Self {
-        Self::new(None, None)
-    }
-
     pub(in crate::high_level_api) fn default_big() -> Self {
         Self {
-            block_parameters: Some(
-                crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS.into(),
-            ),
+            block_parameters: crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS.into(),
             wopbs_block_parameters: None,
         }
     }
 
     pub(in crate::high_level_api) fn default_small() -> Self {
         Self {
-            block_parameters: Some(
-                crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_PBS_KS.into(),
-            ),
+            block_parameters: crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_PBS_KS.into(),
             wopbs_block_parameters: None,
         }
     }
 
     pub fn enable_wopbs(&mut self) {
-        let block_parameter = self
-            .block_parameters
-            .get_or_insert(crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS.into());
```

Comment:
Why was it done this way?
Why change (even if IMO it's better in this PR)?

Reply:
Previously users had to 'enable' the types they wanted to use (FheBool, FheUint2, FheUin3, etc)

So potentially a user could have called the `ConfigBuilder::enable_function_evaluation_integer` without having called `enable_default_integers` before, and in that case, instead of panicking to require calling `enable_default_integers` we chose to set the block parameters ourselves.

But now that there is no choice of which type to enable, there is always a block_parameters